### DISCUSSION TITLE
[Bug]: Azure OpenAI [/assistants/chat/] 400 Invalid type for 'messages[0].content': expected a string, but got an array instead

### DISCUSSION THREAD

**Discussion by pxz2016** [danny-avila/LibreChat#3115] [Troubleshooting]
### What happened?

Suddenly, this error occurred. It was normal before

### Steps to Reproduce

just send a prompt to assitants

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Both Azure Assistants and Assistants are working on my end. Please share a reproduction and relevant logs.

**Reply by pxz2016** [reply]
The assistant in the east us region encountered this error，I have tested several east us endpoints with this situation, starting from yesterday. The assistant just created today is also in the same situation

**Reply by pxz2016** [reply]
What is the azure  region you are testing?

**Reply by zdjohn** [reply]
having similar problems, however seems like a issue on the azure end...

**Reply by zdjohn** [reply]
azure doc seems also not updated https://learn.microsoft.com/en-us/azure/ai-services/openai/assistants-reference-messages?tabs=python#create-message
compare to the openai doc: https://platform.openai.com/docs/api-reference/messages/createMessage

so I wonder if that's Azure being behind again
